content/en/product_analytics/charts/_index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Visualizing with charts
+description: The various Charts help you understand the path your users follow as they discover your product, service, or brand.
+further_reading:
+- link: "/product_analytics/"
+  tag: "Documentation"
+  text: "Product Analytics"
+---
+
+## Overview
+
+Use the various charts to visualize your users' journeys from end-to-end to discover the different ways users navigate your application. You can extract the data to identify frictions in the user journey, measure the success of UI changes and inform your application design decisions. 
+
+## Deciding which chart to use
+
+
+### Pathways diagram
+
+{{< img src="/product_analytics/overview_pathways_ga.png" alt="Use Pathways to visualize all user journeys across your application to analyze the critical path">}}
+
+The [Pathways diagram][3] allows you to visualize all user journeys across your application to identify the most important contributions to a flow.
+
+
+### Funnel analysis
+
+{{< img src="/product_analytics/overview_funnel_ga.png" alt="Understand end-to-end conversions with Funnel Analysis.">}}
+
+With [funnel analysis][2], you can understand the end-to-end conversion of a single essential workflow. You can get a detailed drilldown in the sidepanel to understand why conversation rates are what they are.
```

Comment:
Style guide says to avoid buzzy words like "drilldown":
```suggestion
With [funnel analysis][2], you can understand the end-to-end conversion of a single essential workflow. You can view details in the side panel to understand why conversation rates are what they are.
```